src/dataflow/merge/test.py

```diff
@@ -35,122 +35,132 @@ def extra_var_column_value():
     return "bar"
 
 @pytest.fixture
-def mudata_with_extra_var_column(tmp_path, request, extra_var_column_value):
-    [sample1_path, sample2_path] = request.param
+def extra_var_column_name():
+    return "test"
+
+@pytest.fixture
+def mudata_with_extra_var_column(random_h5mu_path, request, extra_var_column_value, extra_var_column_name):
+    [sample1_path, sample2_path] = request.getfixturevalue(request.param)
     result = []
     for sample_path, column_value in ((sample1_path, extra_var_column_value), (sample2_path, np.nan)):
-        sample = md.read_h5mu(sample_path)
+        sample = read_h5mu(sample_path)
         mod_names = list(sample.mod.keys())
         assert len(mod_names) == 1
         mod_name = mod_names[0]
-        sample.mod[mod_name].var['test'] = column_value
+        sample.mod[mod_name].var[extra_var_column_name] = column_value
         sample.var = sample.var.convert_dtypes(infer_objects=True,
                                                convert_integer=True,
                                                convert_string=False,
                                                convert_boolean=True,
                                                convert_floating=False)
-        new_path = tmp_path / (Path(sample_path).stem + "_extra_col.h5mu")
+        new_path = random_h5mu_path()
         sample.write(new_path)
         result.append(new_path)
     return result
 
 
-def test_merge(run_component):
+def test_merge(run_component, random_h5mu_path, split_small_mudata_path):
```

Comment:
I there a reason not to use `small_mudata_mod1_path` and `small_mudata_mod2_path` here? I would use them directly.